modules/system/oagw/docs/PRD.md

```
@@ -0,0 +1,723 @@
+
+#### Types Registry Contract
+
+- [ ] `p1` - **ID**: `cpt-cf-oagw-contract-types-registry`
+
+- **Direction**: required from client
+- **Protocol/Format**: In-process Rust trait call via `types_registry` SDK
+- **Compatibility**: Must match `types_registry` SDK version in workspace
+
+## 8. Use Cases
+
+#### Proxy HTTP Request
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)